cinema/serializers.py

```diff
@@ -125,9 +125,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
It's better to use the `validate_<field_name>` method for field-specific validation in Django REST Framework. This method is automatically called by DRF when validating the data. If you have a validation that depends on multiple fields, then it's okay to use the general `validate` method.